client/types.ts

```diff
@@ -55,7 +55,8 @@ export interface Theme {
 	theme_uri: string;
 	theme_tier: {
 		slug: string;
-		feature: string;
+		feature?: string;
+		featureList?: string[];
```

Comment:
I see that `feature` here doesn't have the `| null` part, so I'm not sure if we could use the `ThemeTier` type already defined on another file, but it would be better to not have it duplicated..